tests/helpers.py

```diff
@@ -33,6 +33,13 @@ def create_unique_resource_name(resource_description, resource_type):
     return f"{resource_type}-{resource_description[:23]}-{current_date_time[:10]}"
 
 
+def get_crd_dict(path_to_dict):
```

Comment:
As this is not a test helper function, I think it would be better to keep this function in other place like utility.templating

Reply:
done